dcipipeline/pipeline.yml

```diff
@@ -12,7 +13,7 @@
     type: cnf
     location: agents/rh-cnf
     inputs:
-      hosts: agents/rh-cnf/hosts
+      kubeconfig: agents/rh-cnf/kubeconfig
```

Comment:
we should not change it here in the example pipeline because it will make it fail.